components/ui/StartEndCard/StartEndCard.styles.ts

```diff
@@ -0,0 +1,42 @@
+import { theme } from "@/constants/theme";
+import styled from "styled-components/native";
+import { colors, ElipseColors } from "./StartEndCard.types";
+
+interface ElipseProps {
+  elipseColor: ElipseColors;
```

Comment:
Corrigir para "EllipsisProps"

```suggestion
  elipseColor: EllipsisColors;
```